src/containers/app/AppFooter.js

```diff
@@ -5,6 +5,7 @@ import Subscribe from "./Subscribe";
 import backgroundImg from "assets/images/globe.png";
 import discordLogo from "assets/images/Discord-Logo.svg";
 import decent from 'assets/images/decent.png';
+import telegramLogo from 'assets/images/telegram_logo.png';
```

Comment:
how about switching to svg? https://commons.wikimedia.org/wiki/File:Telegram_logo.svg

Reply:
Thanks. I fixed it.